### PR TITLE
[Dev] Database Live Tests should depart from using deprecated CIDatabaseTestCase

### LINKED ISSUE #4351: [Dev] Database Live Tests should depart from using deprecated CIDatabaseTestCase
With the merge of the refactor of `CIDatabaseTestCase`, test cases living in `Database/Live` group should no longer be using it as it is already deprecated in favor of using the `DatabaseTestTrait`. Test cases should now use the trait instead.

### PR DESCRIPTION
**Pull request by Instrye**
Fixes #4351

use `DatabaseTestTrait` instead `CIDatabaseTestCase`

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPdocs
- [x] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Comment by Instrye** [comment]
Hope someone can update the guide,

File: admin/module/tests/_support/DatabaseTestCase.php [hunk 1]
```diff
@@ -1,7 +1,12 @@
 <?php namespace Tests\Support;
 
-class DatabaseTestCase extends \CodeIgniter\Test\CIDatabaseTestCase
+use CodeIgniter\Test\CIUnitTestCase;
+use CodeIgniter\Test\DatabaseTestTrait;
+
+class DatabaseTestCase extends \CodeIgniter\Test\CIUnitTestCase
```
Comment (MGatner) [inline admin/module/tests/_support/DatabaseTestCase.php:6]: It looks like you got most of these but missed this one. 
```suggestion
class DatabaseTestCase extends CIUnitTestCase
```

**Review by MGatner — CHANGES_REQUESTED** [review]
Missed one! This is a great step, User Guide should follow at some point.

**Comment by MGatner** [comment]
@Instrye Please see my comment, and you will need to rebase off the recent Rector changes to **tests/**.

**Pull request closed without merging**